docs/components/connectors/out-of-the-box-connectors/graphql.md

```diff
@@ -0,0 +1,212 @@
+
+![graphql Connector query](../img/connectors-graphql-query.png)
+
+:::note
+Secrets are currently not supported in the Query/Mutation of a GraphQL Connector.
+:::
+
+:::note
+You can test your queries on publicly available GraphQL API here: [https://studio.apollographql.com/public/star-wars-swapi/home?variant=current](https://studio.apollographql.com/public/star-wars-swapi/home?variant=current)
+:::
+
+#### Example
+
+```text
+query Query {
+  allFilms {
+    films {
+      title
+      director
+      releaseDate
+      speciesConnection {
+        species {
+          name
+          classification
+        }
+      }
+    }
+  }
+}
+```
+
+### Variables
+
+You can specify [variables](https://graphql.org/learn/queries/#variables) to your queries/mutations.
+
+The **Variables** field can be configured using the ![feel-icon](../img/feel-icon.png) [FEEL Map](https://camunda.github.io/feel-scala/docs/reference/language-guide/feel-data-types/#context) data type.
+
+```text
+= {
+    "id": "secrets.GRAPHQL_ENTITY_ID",
+    "includeDroids": false,
+}
+```
+
+![graphql Connector variables](../img/connectors-graphql-variables.png)
+
+:::note
+Secrets are not like regular variables and must be wrapped in double quotes (`"`) when used in an expression.
+:::
+
+#### Example
+
+Query:
+
+```text
+query Root($id: ID, $includeGender: Boolean!) {
+  person (id: $id) {
+    name,
+    height,
+    gender @include(if: $includeGender)
+  }
+}
+```
+
+Variables:
+
+```text
+{
+  "id": "cGVvcGxlOjI=",
+  "includeGender": false
+}
+```
+
+## Connection timeout
+
+### Connection timeout
+
+To set connection timeout in your request, set it in seconds in the **Connection Timeout** section.
+This is not a required field, with a default value of 20 seconds. To set an infinite timeout, set this value to `0`.
+
+![graphql connector http request body](../img/connectors-graphql-timeout.png)
+
+## Response Mapping
+
+The HTTP response will be available in a temporary local `response` variable. This variable can be mapped to the process by specifying the **Result Variable**.
+
+The following fields are available in the `response` variable:
+
+- **status**: Response status
+- **body**: Response body of your request
+- **headers**: Response headers
+
+Additionally, you can choose to unpack the content of your `response` into multiple process variables using the ![feel-icon](../img/feel-icon.png) **Result Expression**, which is a [FEEL Context Expression](/components/modeler/feel/language-guide/feel-context-expressions.md).
+
+```text
+= {
+    person: response.body.data.person
+}
+```
+
+![graphql connector http response mapping](../img/connectors-graphql-response-mapping.png)
+
+The next steps in your process will have access to the `graphqlQueryResponse` variable that will contain the full response and the mapped variables from the result expression: `person`.
```

Comment:
Should this not be below the line 194?

Reply:
I tried to be aligned with the rest connector's documentation. That has the same structure, but if you think it would make more sense to place this further up to line 194, I do not have anything against it.

Reply:
I overlooked the screenshot. With the screenshot together it makes sense. Thanks